packages/models-library/src/models_library/api_schemas_webserver/product.py

```diff
@@ -0,0 +1,31 @@
+from decimal import Decimal
+from typing import Any, ClassVar
+
+from pydantic import Field, validator
+
+from ._base import OutputSchema
+
+
+class ProductPriceGet(OutputSchema):
+    product_name: str
+    usd_per_credit: Decimal | None = Field(
```

Comment:
;) you need a NonNegativeDecimal... is there no such thing?

Reply:
true ...